bin/direxec.sh

```diff
@@ -1,21 +0,0 @@
-#!/bin/bash
-# Thanks Camptocamp for the idea!
-# http://www.camptocamp.com/en/actualite/flexible-docker-entrypoints-scripts/
-set -e
-
-mode=$(basename $0 .sh)
-
-for dir in custom common; do
-    dir=/opt/odoo/$dir/$mode.d
-    if [ -d $dir ]; then
-        log INFO Executing contents of $dir
-        run-parts --exit-on-error $dir
-    else
-        log WARNING Skipping execution of $dir: not found
-    fi
-done
-
-if [ -n "$1" ]; then
-    set -x
-    exec "$@"
```

Comment:
I'm still missing this part. The entrypoint is supposed to execute any extra arguments as a command after doing its pre-tasks. Also this `exec "$@"` part makes the subprocess here take the PID of the parent process (which should be 1 in normal run contexts), making all signals work as expected with no extra work. Not sure how to mimic that in Python though, but we need it for sure.